2016-11-06T18:51:49.531Z - error: [gme:server:standalone:mongo] tried to insert existing hash - the two objects were NOT equal  newObject=["Object","__v","1.0.0","_id","#1afb3e18438b0c7ec94e596c2a2a3e91a06a43ab","_meta",["Object","children",["Object","_sets",["Object","_hasownrelation",true,"_nullptr",["Object","_hasownrelation",true,"atr",["Object","name","_null_pointer"]],"items",["Object","k",["Object","_hasownrelation",true,"atr",["Object","max",-1,"min",-1],"reg",["Object","_","_"]],"reg",["Object","_","_"]],"ovr",["Object","",["Object","items","/_nullptr"]]],"reg",["Object","_sets_",4]]],"atr",["Object","_relguid","f951d56f47b106ce20e3eded02e203ca","name","Language"],"ovr",["Object"],"reg",["Object","position",["Object","x",233,"y",228]]], oldObject=["Object","__v","1.0.0","_id","#1afb3e18438b0c7ec94e596c2a2a3e91a06a43ab","_meta",["Object","children",["Object","_sets",["Object","_hasownrelation",true,"_nullptr",["Object","_hasownrelation",true,"atr",["Object","name","_null_pointer"]],"items",["Object","k",["Object","_hasownrelation",true,"atr",["Object","max",-1,"min",-1],"reg",["Object","_","_"]],"reg",["Object","_","_"]],"ovr",["Object","",["Object","items","/_nullptr"]]],"reg",["Object","_sets_",4]]],"atr",["Object","_relguid","f951d56f47b106ce20e3eded02e203ca","name","Language"],"reg",["Object","position",["Object","x",233,"y",228]]]
2016-11-06T18:51:49.544Z - error: [gme:server:standalone:storage] Error: tried to insert existing hash - the two objects were NOT equal #1afb3e18438b0c7ec94e596c2a2a3e91a06a43ab
2016-11-24T00:14:17.506Z - error: [gme:server:standalone] Failed to start server port=8888, code=EADDRINUSE, errno=EADDRINUSE, syscall=listen, address=::, port=8888
2016-11-24T00:14:17.510Z - error: [gme:server:standalone] Error at server start Error: listen EADDRINUSE :::8888
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at Server._listen2 (net.js:1253:14)
    at listen (net.js:1289:10)
    at Server.listen (net.js:1385:5)
    at C:\Users\CYBERTON\Documents\MIC\mic-chariot-project\node_modules\webgme\src\server\standalone.js:221:30
    at _fulfilled (C:\Users\CYBERTON\Documents\MIC\mic-chariot-project\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\CYBERTON\Documents\MIC\mic-chariot-project\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\CYBERTON\Documents\MIC\mic-chariot-project\node_modules\q\q.js:796:13)
    at C:\Users\CYBERTON\Documents\MIC\mic-chariot-project\node_modules\q\q.js:556:49
2016-11-27T19:23:38.748Z - error: [gme:server:worker:simpleworker:pid_42791:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-11-27T19:23:38.911Z - error: [gme:server:worker:simpleworker:pid_42791:WorkerFunctions:PluginManagerBase] The main callback is being called more than once! metadata=Error
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:217:34
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:97:26
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
    at emitOne (events.js:96:13)
2016-11-27T19:23:38.912Z - error: [gme:server:worker:simpleworker:pid_42791:WorkerFunctions] plugin [CHARIOTInterpreter] failed with error Error: The main callback is being called more than once!
    at finish (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/server/worker/workerrequests.js:129:56)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:228:21
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:97:26
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
2016-11-27T19:23:38.918Z - error: [gme:server:worker:simpleworker:pid_42791:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] {"_id":"583b32ba36ed3d27a725ad23","title":"Jaws","year":1975,"director":"Steven Spielberg","rating":"PG"}
2016-11-27T19:40:14.923Z - error: [gme:server:worker:simpleworker:pid_42885:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-11-27T19:40:15.083Z - error: [gme:server:worker:simpleworker:pid_42885:WorkerFunctions:PluginManagerBase] The main callback is being called more than once! metadata=Error
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:217:34
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:95:33
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
    at emitOne (events.js:96:13)
2016-11-27T19:40:15.084Z - error: [gme:server:worker:simpleworker:pid_42885:WorkerFunctions] plugin [CHARIOTInterpreter] failed with error Error: The main callback is being called more than once!
    at finish (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/server/worker/workerrequests.js:129:56)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:228:21
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:95:33
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
2016-11-27T19:40:15.086Z - error: [gme:server:worker:simpleworker:pid_42885:WorkerFunctions:PluginManagerBase] The main callback is being called more than once! metadata=Error
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:217:34
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:95:33
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
    at emitOne (events.js:96:13)
2016-11-27T19:40:15.087Z - error: [gme:server:worker:simpleworker:pid_42885:WorkerFunctions:PluginManagerBase] The main callback is being called more than once! metadata=Error
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:217:34
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:95:33
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
    at emitOne (events.js:96:13)
2016-11-27T19:40:15.088Z - error: [gme:server:worker:simpleworker:pid_42885:WorkerFunctions:PluginManagerBase] The main callback is being called more than once! metadata=Error
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:217:34
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:95:33
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
    at emitOne (events.js:96:13)
2016-11-27T19:40:15.089Z - error: [gme:server:worker:simpleworker:pid_42885:WorkerFunctions:PluginManagerBase] The main callback is being called more than once! metadata=Error
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/src/plugin/managerbase.js:217:34
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:95:33
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/collection/core.js:128:9
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1197:7
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/db.js:1905:9
    at Server.Base._callHandler (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/base.js:453:41)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:488:18
    at MongoReply.parseBody (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
    at .<anonymous> (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/webgme/node_modules/mongodb/lib/mongodb/connection/server.js:446:20)
    at emitOne (events.js:96:13)
2016-11-27T20:03:17.545Z - error: [gme:server:worker:simpleworker:pid_42960:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-11-27T20:23:54.339Z - error: [gme:server:worker:simpleworker:pid_43033:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-11-27T20:45:24.945Z - error: [gme:server:worker:simpleworker:pid_43144:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-11-27T20:49:25.391Z - error: [gme:server:worker:simpleworker:pid_43178:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-11-28T15:46:00.639Z - error: [gme:server:worker:simpleworker:pid_47709:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T18:39:51.276Z - error: [gme:server:worker:simpleworker:pid_21758:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T19:06:19.387Z - error: [gme:server:worker:simpleworker:pid_21758:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T19:08:50.710Z - error: [gme:server:worker:simpleworker:pid_21758:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T19:11:29.660Z - error: [gme:server:worker:simpleworker:pid_23408:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T19:15:14.950Z - error: [gme:server:worker:simpleworker:pid_23408:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T19:47:57.415Z - error: [gme:server:worker:simpleworker:pid_23408:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T19:52:45.306Z - error: [gme:server:worker:simpleworker:pid_23408:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T20:15:44.595Z - error: [gme:server:worker:simpleworker:pid_23408:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-01T20:15:44.598Z - error: [gme:server:worker:simpleworker:pid_23408:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] TypeError: Cannot read property 'split' of undefined
    at CHARIOTInterpreter.getTree (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:203:63)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:67:34
    at _fulfilled (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:796:13)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:604:44
    at runSingle (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:137:13)
    at flush (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
2016-12-01T20:15:44.598Z - error: [gme:server:worker:simpleworker:pid_23408:WorkerFunctions] plugin [CHARIOTInterpreter] failed with error TypeError: Cannot read property 'split' of undefined
    at CHARIOTInterpreter.getTree (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:203:63)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/src/plugins/CHARIOTInterpreter/CHARIOTInterpreter.js:67:34
    at _fulfilled (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:796:13)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:604:44
    at runSingle (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:137:13)
    at flush (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-chariot-project/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
2016-12-01T22:38:57.487Z - error: [gme:server:worker:simpleworker:pid_25394:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-10T03:23:10.443Z - error: [gme:server:worker:simpleworker:pid_8406:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-10T03:44:39.618Z - error: [gme:server:standalone] Failed to start server port=8888, code=EADDRINUSE, errno=EADDRINUSE, syscall=listen, address=::, port=8888
2016-12-10T03:44:39.620Z - error: [gme:server:standalone] Error at server start Error: listen EADDRINUSE :::8888
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at Server._listen2 (net.js:1257:14)
    at listen (net.js:1293:10)
    at Server.listen (net.js:1389:5)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/webgme/src/server/standalone.js:221:30
    at _fulfilled (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:796:13)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:556:49
2016-12-10T03:45:14.647Z - error: [gme:server:standalone] Failed to start server port=8888, code=EADDRINUSE, errno=EADDRINUSE, syscall=listen, address=::, port=8888
2016-12-10T03:45:14.649Z - error: [gme:server:standalone] Error at server start Error: listen EADDRINUSE :::8888
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at Server._listen2 (net.js:1257:14)
    at listen (net.js:1293:10)
    at Server.listen (net.js:1389:5)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/webgme/src/server/standalone.js:221:30
    at _fulfilled (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:796:13)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:556:49
2016-12-10T03:45:56.975Z - error: [gme:server:standalone] Failed to start server port=8888, code=EADDRINUSE, errno=EADDRINUSE, syscall=listen, address=::, port=8888
2016-12-10T03:45:56.977Z - error: [gme:server:standalone] Error at server start Error: listen EADDRINUSE :::8888
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at Server._listen2 (net.js:1257:14)
    at listen (net.js:1293:10)
    at Server.listen (net.js:1389:5)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/webgme/src/server/standalone.js:221:30
    at _fulfilled (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:796:13)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:556:49
2016-12-10T03:47:29.591Z - error: [gme:server:standalone] Failed to start server port=8888, code=EADDRINUSE, errno=EADDRINUSE, syscall=listen, address=::, port=8888
2016-12-10T03:47:29.593Z - error: [gme:server:standalone] Error at server start Error: listen EADDRINUSE :::8888
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at Server._listen2 (net.js:1257:14)
    at listen (net.js:1293:10)
    at Server.listen (net.js:1389:5)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/webgme/src/server/standalone.js:221:30
    at _fulfilled (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:796:13)
    at /Users/chinmaya/Google Drive/Studies/Semesters/CS 6388-1- MIC/Final_Project/mic-project-chariot-webgme/node_modules/q/q.js:556:49
2016-12-10T03:51:12.919Z - error: [gme:server:worker:simpleworker:pid_9487:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


2016-12-10T03:52:37.624Z - error: [gme:server:worker:simpleworker:pid_9545:WorkerFunctions:PluginManagerBase:plugin:CHARIOTInterpreter] 


